go.mod

```diff
@@ -49,7 +49,7 @@ require (
 	go.uber.org/mock v0.5.0
 	go.uber.org/multierr v1.11.0
 	go.uber.org/zap v1.27.0
-	golang.org/x/exp v0.0.0-20240808152545-0cdaa3abc0fa
+	golang.org/x/exp v0.0.0-20240808152545-0cdaa3abc0fa // indirect
```

Comment:
You deleted usage of `golang.org/x/exp`, so it's not a direct dependency anymore. Indirect dependencies should go in the next `require` block

Reply:
strange,
`go mod tidy` didn't want to move it, so I had to move it manually, then use `go mod tidy`. Fixed